client/src/components/History/ContentItem/Dataset/Summary/Ok.vue

```diff
@@ -7,12 +7,12 @@
             <label class="prompt" v-localize>format</label>
             <span class="value">{{ props.dataset.file_ext }}</span>
         </div>
-        <div v-if="props.dataset.metadata_dbkey" class="dbkey">
+        <div v-if="props.dataset.genome_build" class="dbkey">
             <label class="prompt" v-localize>database</label>
-            <span class="value">{{ props.dataset.metadata_dbkey }}</span>
+            <a class="value" @click.stop="listeners['edit']()">{{ props.dataset.genome_build }}</a>
```

Comment:
```suggestion
            <b-link class="value" @click.stop="listeners['edit']()">{{ props.dataset.genome_build }}</b-link>
```

Since, you don't have href on `a` element, it's really hard to understand, that this icon is clickable. So I think, `b-link` would work here better